src/Service/NonCachingIliosFileSystem.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Service;
+
+class NonCachingIliosFileSystem extends IliosFileSystem
+{
+    public function __construct(FilesystemFactory $factory)
+    {
+        $fileSystem = $factory->getNonCachingFilesystem();
+        $fileSystem->disableCache();
```

Comment:
this seems to be the wrong place to invoke `disableCache()`. proposing to move this into `getNonCachingFilesystem()` and to call it there before returning the`$fileSystem` object.

Reply:
Nice! That was a totally invalid invocation left over. Thanks for catching. `getNonCachingFilesystem` doesn't return a class that even has the `disableCache` method.